homeassistant/components/aurora_abb_powerone/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Configure Aurora ABB PowerOne solar PV inverter",
```

Comment:
If you don't define a title, it uses the manifest name, which is in this case the same.
```suggestion
```

Reply:
Done.